common/source/docs/common-advanced-configuration.rst

```diff
@@ -120,6 +120,7 @@ tuning options for the vehicle.
     Transmitter Based Tuning <common-transmitter-tuning>
 [/site]
     UAVCAN Setup <common-uavcan-setup-advanced>
+    Using GPS for Altitude Sensor <common-gps-for-alt>
```

Comment:
I think we should call this page, "GPS for Altitude" and group it with "GPS for Yaw"

Reply:
Will do

Reply:
done